lib/stat_tracker.rb

```diff
@@ -1,9 +1,8 @@
-require 'CSV'
-require './lib/team'
-require './lib/game'
+require 'csv'
+require_relative 'season'
 
 class StatTracker
-  attr_reader :games, :teams, :game_teams
+  attr_reader :games_file, :teams_file, :game_teams_file
```

Comment:
i hope this didn't cause you as much trouble as it caused me. i made those changes in my local files but good to see them here, too